src/cli/cli-report.c

```diff
@@ -811,15 +814,15 @@ static int run_event_chain_real(struct run_event_state *run_state,
 }
 
 /*
- * Run event from chain. Perform the following steps for each event from chain.
- * 1. Terminates a chain run if a backtrace is not usable.
- * 2. Asks for missing settings.
- * 3. Performs review of problem data if event requires review.
- * 4. Runs events commands.
- * 5. Terminates a chain run if any event from the chain requires termination
- *    (i.e. if it existed with EXIT_STOP_EVENT_RUN).
- * 6. Terminates a chain run if any error occurs.
- * 7. Continues with processing of next event.
+ * Run events from a chain. Perform the following steps for each event:
+ * 1. Terminate the chain run if the backtrace is not usable.
```

Comment:
Now there is a mix of definite and indefinite articles.

Reply:
(Talking about terminating runs)

Reply:
I believe you misunderstood me - I’m talking about 1 and 6.

Reply:
Indeed, you are right. Will fix.